src/Hng.Web/Program.cs

```diff
@@ -13,7 +14,25 @@
 builder.Services.AddControllers();
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerDocs();
-builder.Services.AddApplicationConfig();
+builder.Services.AddSwaggerGen(options =>
+{
```

Comment:
@Godhanded  This already exists. Check `AddSwaggerDocs` extension methods in `Hng.Web/Extensions/Documentation.cs`. Modify the existing one if need be.

Reply:
Fixed!